configure.ac

```diff
@@ -138,10 +138,28 @@ AS_IF([test "x$with_libpng" != xno], [
     ])
 ])
 
+# Check for fluidsynth.
+AC_ARG_WITH([fluidsynth],
+AS_HELP_STRING([--without-fluidsynth],
+    [Build without fluidsynth @<:@default=check@:>@]),
+[],
+[
+    [with_fluidsynth=check]
+])
+AS_IF([test "x$with_fluidsynth" != xno], [
+    PKG_CHECK_MODULES(FLUIDSYNTH, fluidsynth >= 2.2.0, [
+        AC_DEFINE([HAVE_FLUIDSYNTH], [1], [fluidsynth installed])
```

Comment:
This looks inconsistent with the cmake configuration. If I understood it right this will not define `HAVE_FLUIDSYNTH` when it's not available but cmake will define it to 0.